docs/guides/tutorials/phoenix_github_oauth.rst

```diff
@@ -637,9 +637,9 @@ will see in the next query, which is defined in the
 Awesome! We're almost done with our application!
 
 As a final step in this tutorial, we will add 2 routes for the web
-application. 1st for redirecting the user to the GitHub OAuth page if it's not
-already logged in and show their username otherwise. And the 2nd one is for
-logging into the application through GitHub.
+application. The first will redirect the user to the GitHub OAuth page if
+they're not already logged in, and show their username otherwise. And the
+second one is for logging into the application through GitHub.
```

Comment:
```suggestion
application. The first will redirect the user to the GitHub OAuth page if
they're not already logged in or will show their username otherwise. The second
is for logging into the application through GitHub.
```